nrf_modem/doc/variant_features.rst

```diff
@@ -3,42 +3,66 @@
 Variants and features
 #####################
 
-The Modem library comes in two variants, each with a specific feature set and support for a specific variant of the nRF91 Series firmware.
+The Modem library comes in two *variants*, each with a specific feature set and support for the equivalent *modem firmware*.
```

Comment:
```suggestion
The Modem library comes in two variants, each with a specific feature set and support for the equivalent modem firmware.
```
I think these need not be in italics as these are normal terms used in the docs.